tests/conftest.py

```diff
@@ -185,3 +185,16 @@ def adata_mapping() -> AnnData:
     adata = adataref.concatenate(adata1, adata2, batch_key="batch", join="outer")
     adata.obs_names_make_unique()
     return adata
+
+
+@pytest.fixture()
+def adata_translation() -> AnnData:
+    rng = np.random.RandomState(31)
+    adatas = [AnnData(X=csr_matrix(rng.normal(size=(100, 60))), dtype=float) for _ in range(3)]
+    adata = adatas[0].concatenate(*adatas[1:], batch_key="batch")
+    adata.obs["celltype"] = rng.choice(["A", "B", "C"], size=len(adata))
+    adata.obs["celltype"] = adata.obs["celltype"].astype("category")
+    adata.layers["counts"] = adata.X.A
```

Comment:
I think the `push` and `pull` functions can't handle sparse data..? At least I get the following error message, when just running the `pull` function on a sparse matrix. (Btw, i get the same error when using the generic `GWProblem`.)
<img width="1158" alt="image" src="https://user-images.githubusercontent.com/114418967/235909696-ae531b50-ca21-49b2-a140-8152d9c1e8e3.png">

Reply:
1. Yes, `push/pull` don't support sparse data, this is due to jax not really handling sparse data nicely yet
2. I think if you want to translate, you should be able to specfiy based on which embedding you want to translate. By default, it should be the embedding which was also used for computing the transport matrix

Reply:
Considering 2: `translate`calls the `push / pull` function depending on `forward` and takes the source or target embedding which was defined in `prepare`, also depending on `forward `. I don't think any other embedinng makes sense here, since the problem is solved for these embeddings. So specifying another embdding to translate would actually would require solving another problem..?

Reply:
I still think it would make sense to apply the matching to any representation of a cell. The better the embedding, the better the alignment also for non-embedded spaces. 

By default I would of course take the embedding based on which the translation coupling was computed